personalized_active_learning/active_learning_flows/base.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         model_output_dim: int,
         model_embedding_dim: int,
         model_hidden_dims: Optional[List[int]] = None,
+        model_dropout: float = 0.2,
```

Comment:
Quick and dirty solution that might become problematic when there will be more model specific params.